src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -303,6 +324,36 @@ export class ExamUpdateComponent implements OnInit {
         );
     }
 
+    validateVisibleDate(isInvalidDate: boolean) {
```

Comment:
Similar faulty naming here.

In some sections, you only set the value. You don't need a function but can do it in the template.
In others you call additional functionality. People reading this functionality must know about the functionality.